Jenkinsfile

```diff
@@ -0,0 +1,56 @@
+fossaCred = [
+    $class: 'StringBinding',
+    credentialsId: 'fossa-api-key',
+    variable: 'FOSSA_API_KEY'
+]
+
+hubCred = [
```

Comment:
Based on comments here: https://github.com/docker/compose/pull/6875#discussion_r323565904,

we want to avoid $class as its legacy and instead follow Nick's suggestion